src/jest/browserMocks.js

```diff
@@ -0,0 +1,20 @@
+var localStorageMock = (function () {
```

Comment:
I know that this is from stackoverflow, however would be nice to get rid of var and refactor it with es6 ;)